src/components/styledSelect/styledSelect.module.css

```diff
@@ -0,0 +1,73 @@
+.root {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  background-color: #fff;
+  font-size: 1rem;
+  margin: auto 0;
+  width: 228px;
+  position: relative;
+}
+
+.header {
+  border: 1px solid #5f5f5f;
+  border-radius: 3px;
+  display: flex;
+  justify-content: space-between;
+  grid-template-columns: 8fr 1fr;
+}
+
+.headerText {
+  font-family: 'Quattrocento Sans', Arial, Helvetica, sans-serif;
+  font-size: 1rem;
+  color: #5f5f5f;
+  padding: 0 8px;
+  margin-top: 8px;
+  margin-bottom: 8px;
+  height: 1rem;
+  line-height: 1;
+}
+
+.trigger {
+  background-color: var(--button-background-color);
+  color: var(--button-text-color);
+  margin: 0;
+  border-left: 1px solid var(--button-border-color);
+  border-right: none;
+  border-top: none;
+  border-bottom: none;
+  cursor: pointer;
+  border-radius: unset;
+  padding: 4px;
+}
+
+.trigger:hover {
+  background-color: var(--button-hover-color);
+}
+
+.dropdown {
+  position: absolute;
+  background-color: #fff;
+  max-height: 107px;
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+  border: 1px solid #5f5f5f;
+  border-top: none;
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  overflow-y: scroll;
+  width: calc(100% - 2px);
```

Comment:
Why ` - 2px`? I think it's because the `border-radius` on the combobox made the dropdown look wider than the combobox when it was at fully `100%` but I'm not totally sure.